### ISSUE TITLE
Escaping issues in 0.6

### ISSUE THREAD

**Issue by yawn** [kreuzwerker/envplate#14]
Example:

```
                <Context path="" docBase="\${catalina.home}/atlassian-jira" reloadable="false" useHttpOnly="true">
```

**Comment by butjar** [comment]
Did you release a new version with escaping included ? For me it looks like escaping is not included in 0.6.

**Comment by yawn** [comment]
Oops, the general commit log is *date-sorted*. In releases it clearly shows 0.6 does *NOT* contain escaping.
